### ISSUE TITLE
VMAS slits

### ISSUE THREAD

**Issue by prjemian** [BCDA-APS/apstools#821]
Add support for [VMAS slits](https://github.com/epics-modules/optics/blob/master/opticsApp/Db/transformVMAS.db) in EPICS Optics repo.

**Comment by prjemian** [comment]
@canismarko : Did you implement this already?

**Comment by canismarko** [comment]
No, i used the 2D HV slits out of the box: https://github.com/spc-group/haven/blob/ion_chamber_beamtime/haven/instrument/slits.py

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by mdwyman** [comment]
The optics modules VMAS slits are supported by the hhl_apertures.py -- could probably close this @prjemian?

**Comment by prjemian** [comment]
@mdwyman Thanks!
